README.md

```diff
@@ -83,15 +83,22 @@ Create a new folder under `packages` if it is a common library or create folder
 
 Make sure to name the package as `@cumulus/package-name`.
 
-## Publishing to NPM
+## Versioning
+
+We use a global versioning approach, meaning version numbers in cumulus are consistent across all packages and tasks, and semantic versioning to track minor, major, and patch version (i.e. 1.0.0). We use Lerna to manage our versioning. Any change will force lerna to increment the version of all packages.
```

Comment:
Nitpicky, but should be "major, minor, and patch"

Reply:
oh good catch